internal/datacoord/garbage_collector.go

```diff
@@ -134,6 +139,16 @@ func (gc *garbageCollector) scan() {
 				v++
 				continue
 			}
+			for _, segID := range segReferIDs {
+				has = strings.Contains(info.Key, strconv.FormatInt(segID, 10))
```

Comment:
nit:

if has = strings.Contains(info.Key, strconv.FormatInt(segID, 10)); has {
	break
}

BTW feels like "has" is not an elegant variable name :)